src/hrg_splittree_eq.h

```diff
@@ -62,7 +62,7 @@
 #ifndef IGRAPH_HRG_SPLITTREE
 #define IGRAPH_HRG_SPLITTREE
 
-#include <iostream>
+#include <string>
 
 using namespace std;
```

Comment:
Another example of `using namespace` in headers that should be avoided. You did not introduce this, but if we are fixing headers anyway, we should probably grep the codebase for `using namespace` and get rid of those that are in headers.